README.md

```diff
@@ -356,6 +356,14 @@ Private channels are created in exactly the same way as normal channels, except
 const channel = socket.subscribe('private-my-channel');
 ```
 
+### Encrypted Channels (BETA)
+
+Like private channels, encrypted channels have their own namespace, 'private-encrypted-'. For more information about encrypted channels, please see the [docs](https://pusher.com/docs/client_api_guide/client_encrypted_channels).
+
+```js
+const channel = socket.subscribe('private-my-channel');
```

Comment:
Should be `private-encrypted-my-channel`

Reply:
🙈 fixed!